src/Sentry.Unity/SentryInitialization.cs

```diff
@@ -26,15 +28,24 @@ public static class SentryInitialization
         internal static LogTimeDebounce LogTimeDebounce = new(TimeSpan.FromSeconds(1));
 
         internal static bool IsInit { get; private set; }
+        private static readonly JsonSerializerOptions _jsonOptions = new()
+        {
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+        };
+
+        private static string SentryOptionsJsonPath => $"Sentry/SentryOptionsJson";
 
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSceneLoad)]
         public static void Init()
         {
-            if (!(Resources.Load("Sentry/SentryOptions") is UnitySentryOptions options))
+            var sentryOptionsTextAsset = Resources.Load<TextAsset>(SentryOptionsJsonPath);

```

Comment:
I wonder if it would make sense to load this at runtime into the `DLL`, as a static string allocation to avoid doing the file I/O in the UI thread slowing down the startup.

Something to consider for the future? https://github.com/getsentry/sentry-unity/issues/82

Reply:
But we need to change it via Unity window. I have no clue for now.

Reply:
The Unity Window will write the asset. But at build time we read the asset and write code :hack-for-perf:.

Future ideas, not to worry until we need to find optimizations for faster startup